src/test/java/com/fasterxml/jackson/databind/type/TestTypeFactoryWithClassLoader.java

```diff
@@ -54,7 +48,7 @@ public void testUsesCorrectClassLoaderWhenThreadClassLoaderIsNull() throws Class
 	verify(spySut).classForName(any(String.class), any(Boolean.class), eq(classLoader));
 	Assert.assertNotNull(clazz);
 	Assert.assertEquals(classLoader, spySut.getClassLoader());
-	Assert.assertEquals(typeModifier,spySut._modifiers[0]);
+//	Assert.assertEquals(typeModifier,spySut._modifiers[0]);
```

Comment:
All assertions such as this one fails, but simply commented out since we are only after classloader. WDYT?

Reply:
@JooHyukKim Yes, I think we are ok wrt not checking `TypeModifier`s: nice to check, not essential.